include/boost/beast/websocket/detail/mask.hpp

```diff
@@ -125,7 +125,7 @@ mask_inplace(boost::asio::mutable_buffer& b, prepared_key& key)
     {
         for(std::size_t i = 0; i < n; ++i)
             p[i] ^= mask[i];
-        rol(key, 4 - n);
```

Comment:
I guess this is causing warnings for 64-bit builds?

Reply:
@vinniefalco That's been my experience with x64 builds on Visual Studio with recent Beast

Reply:
@vinniefalco : correct. I do all Boost testing on 64-bit msvc at /W4 to catch such (and similar) kinds of problems.